.eslintrc

```diff
@@ -87,6 +88,23 @@
         "@shopify/jsx-no-hardcoded-content": "off",
         "@shopify/react-initialize-state": "off"
       }
+    },
+    {
```

Comment:
I was thinking we keep the `polaris-react/polaris-for-vscode/.eslintrc.json` file, remove `"root": true` and other extraneous properties, and extend this root config. This will likely need adjustments, but what do you think about this:

`polaris-react/polaris-for-vscode/.eslintrc.json`
```json
{
  "extends": "../.eslintrc",
  "ignorePatterns": [
    "out",
    "dist",
    "**/*.d.ts"
  ] 
}
```

Reply:
@alex-page when I remove/move the plugin eslint config eslint starts to complain, and we lose auto-fix issues/formatting. Not sure why but using the root level .eslintrc doesn't seem to be doing what we want 🤔 

Extending the root config and keeping the plugin eslintrc as @aaronccasanova does seem to work as expected. I'll go with that unless there are any objections or concerns with that approach?

Reply:
Because auto-fixing stopped, I feel like it wasn't correctly inheriting the root level.